src/Illuminate/Database/Connection.php

```diff
@@ -282,6 +283,18 @@ protected function getDefaultSchemaGrammar()
         //
     }
 
+    /**
+     * Get the schema state for the connection.
+     *
+     * @param  \Illuminate\Filesystem\Filesystem|null  $files
+     * @param  callable|null  $processFactory
+     * @return \Illuminate\Database\Schema\SchemaState|null
+     */
+    public function getSchemaState(Filesystem $files = null, callable $processFactory = null)
+    {
+        //
```

Comment:
```suggestion
        throw new BadMethodCallException('The database driver does not schema state feature.');
```

Also you should add `use` import statement

Reply:
I could do this, however, you don't see an exception being thrown in the stubs above this method